contracts/governance_admin/src/execute/post_proposal_creation_callback.rs

```diff
@@ -103,13 +103,15 @@ pub fn execute_post_governance_proposal_creation_callback(
             )?;
 
             event = event
-                .add_attribute(
-                    "reward_schedules_creation_request_id",
-                    request_id.to_string(),
-                )
+                .add_attribute("request_id", request_id.to_string())
                 .add_attribute("proposal_id", latest_proposal.id.to_string());
         }
     }
 
+    event = event.add_attribute(
+        "proposal_type",
+        serde_json_wasm::to_string(&proposal_type).unwrap(),
```

Comment:
If we are indexing this already then, we don't need to index the request_id too since it would contain it.